tasks/verifySpokePool.ts

```diff
@@ -64,7 +64,11 @@ task("verify-spokepool", "Verify the configuration of a deployed SpokePool")
     console.log("SpokePool.fillDeadlineBuffer()".padEnd(TEXT_PADDING) + ": " + fillDeadlineBuffer);
 
     const wrappedNative = bytes32ToAddress(results.wrappedNativeToken);
-    const wrappedNativeSymbol = spokeChainId === POLYGON ? "WMATIC" : "WETH";
+    const customWrappedNativeSymbols = {
+      [POLYGON]: "WMATIC",
+      [ALEPH_ZERO]: "WAZERO",
+    };
```

Comment:
@melisaguevara We can btw souce this directly from the `PRODUCTION_NETWORKS` object exported by `@across-protocol/constants`. It can be resolved by `PRODUCTION_NETWORKS[ALEPH_ZERO].nativeToken`. It might be worth doing that so we don't need to update this with future tokens.

Reply:
yes that would be definitely better! Let me update this.

Reply:
@pxrl The `PRODUCTION_NETWORKS` object currently only provides the native token, but we actually need the wrapped symbol. Do you think it would make sense to add that in as well?

Reply:
Ah, that's a good point. For now I'd probably be inclined to just assume that any wrapped variant would be prepended with "W" and add that locally, and reserve the right to be wrong about it in future. wdyt?

Reply:
Yes, I think that will work! I implemented it here [96f5c43](https://github.com/across-protocol/contracts/pull/706/commits/96f5c43e595b3763133dee8d4b7f389d44ea5df5)